### PR TITLE
Simplify admin search

### PR DESCRIPTION
**Pull request by noirbizarre**
This pull-request make the admin search more readable.

Search with results before:
Search with result before

After:
Search with result after

Search without any results before:
Search with result before

After:
Search with result after

### CODE REVIEW

File: js/views/search.vue [hunk 1]
```diff
@@ -37,29 +37,38 @@ export default {
         discussions: require('components/discussions/list.vue'),
         Layout
     },
+    computed: {
+        title() {
+            return [
+                this._('Search in your data'),
+                this.$route.query.q
+            ].join(': ');
```
Comment (davidbgk) [inline js/views/search.vue]: You need a translation here for the `:` (takes a non-breaking space before in French).

BTW, I don't why you join and not use a simple concatenation or maybe a template string?

File: js/views/search.vue [hunk 2]
```diff
@@ -37,29 +37,38 @@ export default {
...
+        },
+        no_results() {
+            const collections = [this.datasets, this.communities, this.reuses, this.issues, this.discussions];
+            return !collections.some(function(collection) {
```
Comment (davidbgk) [inline js/views/search.vue]: Batteries included you said? :trollface:

**Comment by davidbgk** [comment]
Way better, thanks. r+

File: js/views/search.vue (hunk 2 above)
Comment (noirbizarre) [inline js/views/search.vue]: This one is great. Equivalent to python `any()`

**Pull request merged**